_work/TGP_work/_test/test_object_lookup_2023-07-25-1746/test_utilities_herbert/test_object_lookup_internals.m

```diff
@@ -36,19 +36,19 @@ function test_parse_split_3 (~)
         function test_parse_split_4 (~)
```

Comment:
Still have bad names

Reply:
Aargh. I must have forgotten to delete this when I dumped the checks in the \work\tgp area when I disinfected the dodgy tests. I'll delete it when I make my next pull request.